pages/content/amp-dev/documentation/guides-and-tutorials/learn/email_fundamentals.md

```diff
@@ -0,0 +1,298 @@
+---
+$title: AMP for Email Fundamentals 
+$order: 1
+author: CrystalOnScript
+formats:
+  - email
+---
+
+If you're familiar with AMP, great news! AMP for Emails is just a subsect of the AMP HTML library. If you're unfamiliar with AMP, also great news! This guide will give you everything you need to know to get started writing valid AMP Emails! 
```

Comment:
Is subsect the right word here? The only meaning I could find is [this one](https://www.merriam-webster.com/dictionary/subsect), but this only refers to sects (as in cults). Imo subset would be a better term.

Reply:
Lol, good catch. I can't believe that slipped past me... twice....